pdc/settings_local.py.dist

```diff
@@ -84,6 +84,9 @@ EMAIL_SUBJECT_PREFIX = '[PDC]'
 # Omit sending e-mails about big data change when author is in this whitelist.
 #BIG_CHANGE_AUTHORS_WHITELIST = ('some_user',)
 
+# Set the key_id of sigkey "redhatrelease2" as default
```

Comment:
It's not a `key_id`, but name of the key. Also maybe a dummy value would be better as an example (e.g. `Gold Key` or something)?

Reply:
Here will get the Sigkey object by the default key name, this is also set in et side.
And the name is also unique and not null. And we should keep the sigkey table with the default key_id/name before create a release.